upstairs/src/lib.rs

```diff
@@ -5721,164 +5722,161 @@ impl Upstairs {
              * downstairs out, forget any activation requests, and the
              * upstairs goes back to waiting for another activation request.
              */
-            match self.collate_downstairs(&mut ds).await {
-                Ok(res) => {
-                    need_repair = res;
-                    failed_collate = false;
-                }
-                Err(e) => {
-                    need_repair = false;
-                    failed_collate = true;
-                    error!(self.log, "Failed collate with {}", e);
-                }
-            }
-            repair_commands = ds.reconcile_task_list.len();
```

Comment:
Yes, this is no longer here, it's moved down below to the one place where we use it.